include/SFML/Graphics/Rect.hpp

```diff
@@ -146,6 +146,26 @@ class Rect
     bool intersects(const Rect<T>& rectangle, Rect<T>& intersection) const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Get the position of the rectangle
+    ///
+    /// \return Position of rectangle
```

Comment:
I would explicitly mention that this is the position of the top-left corner of the rectangle, just like how it is done in the constructor.